Sources/ArcGISToolkit/Components/Authentication/CertificatePickerViewModifier.swift

```diff
@@ -71,14 +71,18 @@ import ArcGIS
             preconditionFailure()
         }
         
-        Task {
+        Task.detached {
             do {
-                challenge.resume(with: .continueWithCredential(try .certificate(at: certificateURL, password: password)))
+                guard certificateURL.startAccessingSecurityScopedResource() else {
+                    self.showCertificateImportError(nil)
```

Comment:
This will end up showing "The certificate file or password was invalid." Is that ok or should it show some error about permission to access the file if this fails?

Reply:
ok, we can get this in and then follow up with a PR to make better error messages.